modules/cnv-disabling-guest-memory-overhead-accounting.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-managing-guest-memory.adoc
+
+[id="cnv-disabling-guest-memory-overhead-accounting_{context}"]
+= Disabling guest memory overhead accounting
+
+[WARNING]
+====
+This procedure is only useful in certain use-cases and should
+only be attempted by advanced users.
+====
+
+A small amount of memory is requested by each virtual machine instance in
+addition to the amount that you request. This additional memory is used for
+the infrastructure wrapping each `VirtualMachineInstance` process.
+
+Though it is not usually advisable, it is possible to increase the VMI
+density on the node by disabling guest memory overhead accounting.
+
+// .Prerequisites
+
+// *
+
+.Procedure
+
+* To disable guest memory overhead accounting, edit the YAML configuration
```

Comment:
Same here. If it's possible to provide a little more information about this file, please do.